python/tests/sql_aggregates/test_bit_xor.py

```diff
@@ -0,0 +1,127 @@
+        table_name = "bit_xor_distinct"
+        view_query = f'''CREATE VIEW bit_xor_view AS SELECT
+                            BIT_XOR(DISTINCT c1) AS c1, BIT_XOR(DISTINCT c2) AS c2, BIT_XOR(DISTINCT c3) AS c3, BIT_XOR(DISTINCT c4) AS c4, BIT_XOR(DISTINCT c5) AS c5, BIT_XOR(DISTINCT c6) AS c6, BIT_XOR(DISTINCT c7) AS c7, BIT_XOR(DISTINCT c8) AS c8
+                        FROM {table_name}'''   
+        self.execute_query(pipeline_name, expected_data, table_name, view_query) 
+
+class Bit_XOR_Distinct_Groupby(TestAggregatesBase):
+    def test_bit_xor_distinct_groupby(self):
+        pipeline_name = "test_bit_xor_distinct_groupby"
+        # validated using postgres
+        new_data = [
+            {"id" :0 ,"c1": 4, "c2": 2, "c3": 30, "c4": 14, "c5": None, "c6": 60, "c7": 70, "c8": 18},
+            {"id": 1,"c1": 5, "c2": 3, "c3": None, "c4": 9, "c5": 51, "c6": 6, "c7": 72, "c8": 2}]
+        self.add_data(new_data)
+        expected_data = [{'id': 0, 'c1': 1, 'c2': 2, 'c3': 30, 'c4': 10, 'c5': 5, 'c6': 58, 'c7': 70, 'c8': 26}, 
+                         {'id': 1, 'c1': 1, 'c2': 3, 'c3': 4, 'c4': 15, 'c5': 49, 'c6': 5, 'c7': 76, 'c8': 2}]
+        table_name = "bit_or_distinct_groupby"
+        view_query = f'''CREATE VIEW bit_xor_view AS SELECT
+                            id, BIT_XOR(DISTINCT c1) AS c1, BIT_XOR(DISTINCT c2) AS c2, BIT_XOR(DISTINCT c3) AS c3, BIT_XOR(DISTINCT c4) AS c4, BIT_XOR(DISTINCT c5) AS c5, BIT_XOR(DISTINCT c6) AS c6, BIT_XOR(DISTINCT c7) AS c7, BIT_XOR(DISTINCT c8) AS c8
+                        FROM {table_name}
+                        GROUP BY id;'''
+        self.execute_query(pipeline_name, expected_data, table_name, view_query) 
+
+class Bit_XOR_Where(TestAggregatesBase):  
+    def test_bit_xor_where(self):
+        pipeline_name ="test_bit_xor_where"
+        # validated using postgres
+        expected_data = [{'c1': 4, 'c2': 3, 'c3': 4.0, 'c4': 6, 'c5': 2, 'c6': 3, 'c7': 4.0, 'c8': 2},
+                         {'c1': 5, 'c2': 2, 'c3': None, 'c4': 4, 'c5': 5, 'c6': 6, 'c7': None, 'c8': 8}]
+        table_name = "bit_xor_where"
+        view_query = f'''CREATE VIEW bit_xor_view AS
+                        WITH bit_xor_val AS(
+                            SELECT
+                                BIT_XOR(c1) AS bit_xor_c1, BIT_XOR(c2) AS bit_xor_c2, BIT_XOR(c3) AS bit_xor_c3, BIT_XOR(c4) AS bit_xor_c4, BIT_XOR(c5) AS bit_xor_c5, BIT_XOR(c6) AS bit_xor_c6, BIT_XOR(c7) AS bit_xor_c7, BIT_XOR(c8) AS bit_xor_c8
+                            FROM {table_name})
+                            SELECT t.c1, t.c2, t.c3, t.c4, t.c5, t.c6, t.c7, t.c8
+                            FROM {table_name} t
+                            WHERE (t.c8) < (SELECT bit_xor_c8 FROM bit_xor_val);'''   
+        self.execute_query(pipeline_name, expected_data, table_name, view_query)
+
+class BIT_XOR_Where_Groupby(TestAggregatesBase):       
```

Comment:
for xor can you add a test where the input has 2 indentical and 3 identical rows?
with and without distinct